user/cinnamon/template.py

```diff
@@ -0,0 +1,89 @@
+pkgname = "cinnamon"
+pkgver = "6.2.9"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Ddefault_library=shared"]
+hostmakedepends = [
+    "gobject-introspection",
+    "intltool",
+    "meson",
+    "pkgconf",
+]
+makedepends = [
+    "at-spi2-core-devel",
+    "cinnamon-menus-devel",
+    "cjs-devel",
+    "dbus-devel",
+    "glib-devel",
+    "gstreamer-devel",
+    "gtk+3-devel",
+    "libx11-devel",
+    "libxml2-devel",
+    "mesa-devel",
+    "muffin-devel",
+    "networkmanager-devel",
+    "polkit-devel",
+    "xapp-devel",
+]
+depends = [
+    "bash",
+    "caribou",
+    "cinnamon-control-center",
+    "cinnamon-screensaver",
+    "cinnamon-session",
+    "libtimezonemap",
+    "mint-themes",
+    "nemo",
+    "network-manager-applet",
+    "python-pam",
+    "python-pexpect",
+    "python-pillow",
+    "python-pytz",
+    "python-requests",
+    "python-tinycss2",
+    "touchegg",
+]
+pkgdesc = "Linux desktop that provides a traditional user experience"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://projects.linuxmint.com/cinnamon"
+source = (
+    f"https://github.com/linuxmint/cinnamon/archive/refs/tags/{pkgver}.tar.gz"
+)
+sha256 = "a0b15b98a899532d531689ea11ff6805a87fcf5994e238322b6c916bf8c29919"
+broken_symlinks = ["etc/xdg/menus/cinnamon-applications-merged"]
+# meson.build: tests are not currently functional
+options = ["!check", "!cross"]
+
+
+@subpackage("cinnamon-apps")
+def _(self):
+    self.subdesc = "apps"
+    self.install_if = [self.parent]
+    self.depends = ["gnome-terminal", "xapp-apps"]
```

Comment:
is pulling in gnome-terminal something that upstream encourages or suggests? if not, then i'm not quite sure we should do it

Reply:
Yes.
1. By default, the GNOME Terminal application is pinned to the taskbar
2. mint-y-icons has an icon for GNOME Terminal but not GNOME Console.
3. Linux Mint uses GNOME Terminal because it uses GTK+3, while GNOME Console uses GTK4 and libadwaita (which it avoids because libadwaita doesn't respect themes, see https://blog.linuxmint.com/?p=4675 ).